### PR TITLE
Create Badge Component

### PR DESCRIPTION
**Pull request by khyatibaria**
### Create Badge Component
Please Review
image

### CODE REVIEW

File: badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+}
+.badge-count {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  background-color: #51fc93;
```
Comment (yashpatel1797) [inline badge/badge.css]: if you are using the HEX method then use It everywhere.

File: badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            <div class="badge-count badge-xl badge-color">10</div>
+        </figure>
+
+        <figure class="icon-badge">
+            <img class="img-responsive " src="./assets/cart.svg" />
+            <div class="icon-badge-count" id="icon">5</div>
+        </figure>
+        <figure class="icon-badge">
+            <img class="img-responsive " src="./assets/message.svg" />
+            <div class="icon-badge-count" id="icon">5</div>
```
Comment (yashpatel1797) [inline badge/badge.html]: Avoid using the same id for different element if want then gives class.

**Review by yashpatel1797 — CHANGES_REQUESTED** [review]
just two suggestions. Even you can declare variables for color then use it everywhere.

File: badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link href="badge.css" rel="stylesheet" type="text/css" />
+    <script src="https://kit.fontawesome.com/99b4ab4b7a.js" crossorigin="anonymous"></script>
```
Comment (bharati-21) [inline badge/badge.html:11]: You can also add meta tags for author and description.

**Review by bharati-21 — APPROVED** [review]
I agree with the points @yashpatel1797 mentioned in their PR review. 
But apart from that, everything looks good to me!

**Pull request merged**